cartographer/sensor/compressed_point_cloud.cc

```diff
@@ -158,16 +149,15 @@ void CompressedPointCloud::ConstIterator::ReadNextPoint() {
 }
 
 CompressedPointCloud::CompressedPointCloud(const PointCloud& point_cloud)
-    : point_data_(Compress(point_cloud)), num_points_(point_cloud.size()) {}
+    : point_data_(DoCompress(point_cloud)), num_points_(point_cloud.size()) {}
```

Comment:
Didn't do the inline because `point_data_` is const std::vector<>, which is semantically nicer.